### ISSUE TITLE
HTTP 500 error due to getMaxCompanyName related issues

### ISSUE THREAD

**Issue by vanmelick** [myparcelnl/magento#360]
I found 2 related issues in version 2.5.6. Both cause this error:
Sales => Orders => open order => Print Label => HTTP 500 error.

Issue 1:
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to MyParcelNL\Magento\Model\Source\DefaultOptions::getMaxCompanyName() must be of the type string, null given, called in vendor/myparcelnl/magento/Model/Sales/MyParcelTrackTrace.php on line 143 and defined in vendor/myparcelnl/magento/Model/Source/DefaultOptions.php:86

This happens for addresses where the company name is not filled out by the customer, causing getMaxCompanyName() to be called with parameter NULL.

Issue 2: 
PHP message: PHP Fatal error:  Uncaught Error: Undefined class constant 'COMPANY_NAME_MAX_LENGTH' in vendor/myparcelnl/magento/Model/Source/DefaultOptions.php:88

Problem started with commits c0b2a0374ea55d39f65c3604c3d096d49f9ea66a and a28b85464cfd39775266b6351d8d7d22be7cdc08

**Comment by gerben86** [comment]
I'm using the same code, but didn't encounter this.
But implementing a null-check would solve the issue, isn't?
So replacing
```php
if (strlen($address) >= self::COMPANY_NAME_MAX_LENGTH) {
```
with
```php
if ($address != null && (strlen($address) >= self::COMPANY_NAME_MAX_LENGTH)) {
```
should solve the issue.

**Comment by RichardPerdaan** [comment]
Fixed in [4.1.2](https://github.com/myparcelnl/magento/releases/tag/4.1.2) of the MyParcel Magento 2 plugin
